interfaces/sourcegen/src/sourcegen/yaml/templates.yaml

```diff
@@ -17,8 +17,8 @@ yaml-definition: |-
     {% if c_func.returns %}
     {{ 'returns: ' + c_func.returns }}
     {% endif %}
-    {% if implements %}
-    implements: {{ implements }}
+    {% if wraps %}
+    wraps: {{ wraps }}
```

Comment:
I couldn't quite follow how this ends up as an enumerated list in the output, but given that `wraps` is always a single item, could we get it formatted in the output as just
```
Wraps C++ getter: `size_t Phase::nSpecies()`
```
instead of 
```
Wraps C++ getter:
- `size_t Phase::nSpecies()`
```
to make things a little more compact (particularly in the Doxygen HTML output, which is already pretty inefficient at using vertical space).